etc/iscsid.conf

```diff
@@ -347,6 +347,18 @@ node.session.nr_sessions = 1
 # recovery was limited to about five minutes.
 node.session.reopen_max = 0
 
+# When the iscsi targets disappear due to any reason like ports down or
+# the network issues or any other reason, existing sessions will be retried
+# by the iscsi daemon and each retry will have a 3 seconds gap by default.
+# So, the session reopen logs are logged into /var/log/messages for every
+# 3 seconds and due to this messages file is blown out. To avoid this blow
+# out of messages file, uncomment the following config parameter and change
+# the value from 1 to a desirable value like 2 or 3 or 4 and restart
+# iscsid. If the value for this config value is 4, then these session
+# reopen logs will be logged for every 12 seconds (4 * 3 seconds = 12
+# seconds).
+# node.session.reopen_log_freq = 1
+
```

Comment:
Many modern linux systems (like tumbleweed) do not use /var/log/messages, but instead use the systemd journal. Perhaps you can word your description here more generically, something like "session reopen logging occurs every 3 seconds, and can overwhelm the logging system>"?